Add new installation instructions and guides

edwardhorsford opened the pull request:
For consideration / reading.

This PR updates the readme / docs to be more focused on helping new users get started with the kit.

* Updated main `readme`.
* Guide for installing the kit
* Guide for doing first commit
* Guide for pushing to heroku

timpaul commented:
I really like it and think we should merge ASAP, so we can tell the next round of trainees to download the kit. We'll soon learn how effective it is.

edwardhorsford commented:
I've removed the DO NOT MERGE banner. I've also removed the extra `deploying.md` guide. For the moment, I've not included the section on deploying multiple versions, which feels like a further guide to me.

alextea commented:
Next time can we try and have a cleaner commit history? I can help with rebasing and tidying up commits if needed.

edwardhorsford commented:
@alex perhaps you could give me a tutorial? I'd quite like to learn how to do that better. I have another PR that has a lot of commits.

alextea commented:
@edwardhorsford sure, was going to suggest Friday, but we have the training day, so maybe next week?

edwardhorsford commented:
Thanks @alextea that would be great.
